waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -1,31 +1,35 @@
 {.push raises: [Defect].}
 
 import
-  std/tables,
+  std/[tables, deques],
   options, chronos, stint,
   web3,
   eth/keys,
   libp2p/protobuf/minprotobuf,
   stew/arrayops,
   ../../utils/protobuf
 
+const ACCEPTABLE_ROOT_WINDOW_SIZE* = 5
```

Comment:
We are indeed very inconsistent, but are converging to a future where we use `PascalCase` for constants. Feel free to address in this PR or to address collectively in a follow-up PR focused on refactoring. :)

Reply:
Addressed in [e6da347](https://github.com/status-im/nwaku/pull/1177/commits/e6da347a50c7a42eee03bcd89b19a546b8d73d47)